api/core/v2/adhoc.proto

```diff
@@ -24,5 +27,5 @@ message AdhocRequest {
   string reason = 4 [(gogoproto.nullable) = true];
 
   // Metadata contains the name, namespace, labels and annotations of the AdhocCheck
-  ObjectMeta metadata = 5 [(gogoproto.jsontag) = "metadata", (gogoproto.embed) = true, (gogoproto.nullable) = false];
+  ObjectMeta metadata = 5 [(gogoproto.jsontag) = "metadata,omitempty", (gogoproto.embed) = true, (gogoproto.nullable) = false];
```

Comment:
Are we sure that `omitempty` is not going to cause subtle breakage somewhere else?

Reply:
See my comment above!